### PR TITLE
UofT-DSI | LCR - Assignment 2

### PR DESCRIPTION
**Pull request by juliosocher**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Pushing the answers for Assingment-2 of the LCR module.

## What did you learn from the changes you have made?
That the train/test size matters to the result.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
I tried to change the test size value up and down and noticed that reducing the size improved the results, I'm sure this is based on the size of the dataset as we should have enough training data for a good result.

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
No.

## How were these changes tested?
Coding.

## A reference to a related issue in your repository (if applicable)
N/A

## Checklist
- [X] I can confirm that my changes are working as intended

### CODE REVIEW

File: 02_activities/assignments/assignment_2.ipynb
Comment (Aditya-k-23) [inline 02_activities/assignments/assignment_2.ipynb:1]: Cannot do `lm.fit()` on the whole y, we need to use the training set for training and then test it against the test set.

image

**Review by Aditya-k-23 — CHANGES_REQUESTED** [review]

File: 02_activities/assignments/assignment_2.ipynb
Comment (juliosocher) [reply 02_activities/assignments/assignment_2.ipynb:1]: @Aditya-k-23 Thanks for the feedback, so basically changing the numeric_predictors to X_train instead of X, and the y in the fit function to y_train, right?

Comment (Aditya-k-23) [reply 02_activities/assignments/assignment_2.ipynb:1]: Yes

**Review by Aditya-k-23 — APPROVED** [review]
